style.css

```diff
@@ -17,6 +17,17 @@ a {
   margin: 1rem;
   display: flex;
   width: 225px;
+  display: flex;
```

Comment:
The display flex are duplicated. When you add the "flex-direction: column;" at the class flag broke every vertical flags.